media-sound/qtractor: bump to 0.8.4

simonvanderveldt opened the pull request.

a17r commented:
Please use the opportunity to close this bug: https://bugs.gentoo.org/631914

simonvanderveldt commented:
> Please use the opportunity to close this bug: https://bugs.gentoo.org/631914

Unfortunately that would make my setup as a whole useless to me, since enabling Qt5 on Qtractor means one has to enable Qt5 on all (lv2) plugins as well (otherwise Qtractor crashes).
But since the released versions of suil only don't support Qt5 inside gtk that would make those plugins useless in suil based hosts (like jalv).

Also upstream still supports Qt4. Why not just keep the option?

a17r commented:
Keeping Qt4 is no sustainable option: https://bugs.gentoo.org/show_bug.cgi?id=qt4-removal

a17r commented on media-sound/qtractor/qtractor-0.8.4.ebuild:
```diff
@@ -0,0 +1,77 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=5
```
it would be nice to bump this to EAPI 6 as well, looks easy enough

a17r commented on media-sound/qtractor/qtractor-0.8.4.ebuild:
```diff
@@ -0,0 +1,77 @@
+
+inherit qmake-utils flag-o-matic eutils
+
+DESCRIPTION="Qtractor is an Audio/MIDI multi-track sequencer"
```
description should not duplicate package name

a17r commented on media-sound/qtractor/qtractor-0.8.4.ebuild:
```diff
@@ -0,0 +1,77 @@
+	virtual/pkgconfig
+	qt5? ( dev-qt/linguist-tools:5 )"
+
+DOCS="README ChangeLog TODO AUTHORS"
```
should ideally be an array, not string

a17r commented on media-sound/qtractor/qtractor-0.8.4.ebuild:
```diff
@@ -0,0 +1,77 @@
+
+EAPI=5
+
+inherit qmake-utils flag-o-matic eutils
```
eutils can be dropped by bumping to EAPI 6

simonvanderveldt commented:
@a17r I've addressed your comments regarding the ebuild

> media-libs/suil-0.8.4 appears to have USE qt5. If that does not suffice, would you be willing to try out some suil snapshot from git master that does have support?

I'd prefer to wait for a suil release, I can ping drobilla if/when that will happen.
Qtractor's Qt4 build works perfectly fine, imho there's no need to nuke Qt4 support for it right now.

a17r commented:
> I can ping drobilla if/when that will happen.

Please do that. Qtractor can support Qt4 as long as they wish, we won't care. Qt4 is a huge liability by now and future support to keep it building and built against is a pain.
